includes/admin/home/tasks/task/class-sensei-home-task-create-first-course.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * File containing the Sensei_Home_Task_Create_First_Course class.
+ *
+ * @package sensei-lms
+ * @since $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Sensei_Home_Task_Create_First_Course class.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_Home_Task_Create_First_Course implements Sensei_Home_Task {
+	/**
+	 * The ID for the task.
+	 *
+	 * @return string
+	 */
+	public function get_id(): string {
+		return 'create-first-course';
+	}
+
+	/**
+	 * Number used to sort in frontend.
+	 *
+	 * @return int
+	 */
+	public function get_priority(): int {
+		return 200;
```

Comment:
@aaronfc I saw that the Configure Learning Mode task you created also has priority = 200, which means that it should be more or less in the same order. From what I see, according to the tasks, the Create Your First Course step should come before, though.

WDYT about changing the priority of the Configure Learning Mode to 300?

Reply:
Update: I decided to change that in 8e3fec0fa3d129bb4cf1dae275096b8df85e9696 :)